benchmarks/comparison.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+import shutil
+import subprocess
+import tempfile
+
+from django.conf import settings
+
+compare_script = os.path.join(os.path.dirname(__file__), '../ext/art-testing/compare.py')
+
+def render_comparison(results_before, results_after):
+    tmpdir = export_data(results_before, results_after)
+    # FIXME: sandbox this!
+    output = subprocess.check_output([
```

Comment:
I thinkn we do need to migrate to python3 at some point, but  that tends to be a reasonable undertaking and I'm not sure we want to do that right now.

besides this way it's easy to plug any other comparison strategies since we don't rely on them being written in python :)